db/migrate/20200624170002_update_set_type_to_teacher_sets.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UpdateSetTypeToTeacherSets < ActiveRecord::Migration[5.0]
+  def change
+    TeacherSet.find_each do |ts|
+      set_type = ts.get_set_type(ts.set_type)
```

Comment:
let's also provide what to do if the set type field is either null or a value that doesn't conform

Reply:
@daryanypl This method is only to convert 'multi' and 'single' values. not for nil values.
If we want to fix set_type values with out nil we need to update BIB ids from sierra or with back ground methods.